src/test/java/guiTest/ActivateEmailTest.java

```diff
@@ -0,0 +1,34 @@
+package guiTest;
+
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import pages.ActivateEmail;
+import pages.Registration;
+import utils.FunctionalTest;
+
+import static org.junit.Assert.*;
+import static utils.Constants.SIGN_UP_URL;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+
+public class ActivateEmailTest extends FunctionalTest {
+
+    Registration registration = new Registration(driver);
+    ActivateEmail activationEmail = new ActivateEmail(driver);
+
+    @Before
+    public void init() {
+        driver.get(SIGN_UP_URL);
+    }
+
```

Comment:
Should be a part of test06 of the RegistrationTest, just check there if the correct message is displayed.

Reply:
Agreed. Test01 is a part of test06 of the RegistrationTest.